salt/cloud/clouds/xen.py

```diff
@@ -150,20 +151,27 @@ def _get_session():
         __opts__,
         search_global=False
     )
-    ignore_ssl = config.get_cloud_config_value(
```

Comment:
Why are you removing the check for the `ignore_ssl` option here? It seems to me that this shouldn't be changed since the user can specify this value in their cloud configs.

Reply:
I've put that back
please note that settings "ignore_ssl" to "True" failes in my environment
"False" works fine.
this is why I orignialy removed that.

I'll try to submit a fix to that when I have some time.

Reply:
Ah ok, that sounds great if you could submit a fix for that when you have a moment. Thanks! I'll try to get this merged in today, pending tests.